analyses/molecular-subtyping-HGG/02-HGG-molecular-subtyping-subset-files.R

```diff
@@ -79,7 +87,22 @@ gistic_df <-
       "broad_values_by_arm.txt"
     ),
     exdir = file.path(root_dir, "scratch")
-  ))
+  ), data.table = FALSE)
+
+# Read in snv consensus mutation data
+snv_maf_df <-
+  data.table::fread(file.path(root_dir,
+                              "data",
+                              "pbta-snv-consensus-mutation.maf.tsv.gz"),
+                    select = c("Chromosome",
```

Comment:
Is this the subsetting you were referring to in the list? Is this just to save on memory usage?

Reply:
We were not generating any HGG-specific SNV file before. That's what I was referring to in the list. This step is to save on memory though, that's correct.